test/EFCore.PG.FunctionalTests/Query/ArrayQueryTest.cs

```diff
@@ -16,7 +16,7 @@ public ArrayQueryTest(ArrayQueryFixture fixture, ITestOutputHelper testOutputHel
         {
             Fixture = fixture;
             Fixture.TestSqlLoggerFactory.Clear();
-            //Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
+            Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
```

Comment:
Needs to be removed for merging.

Reply:
What is it for? Why commented?

Reply:
This makes a log of debugging information (expression tree, generated SQL) outputted to stdout. It's useful when working on tests, but considerably pollutes the CI logs, so we usually keep it commented out. This is the pattern of work used at EF Core.

Reply:
Is there any way to keep it, but not as a comment? Maybe an environment variable or something else?

Reply:
We could think of something, but it would have to happen in EF Core and not in our provider. I think it's just not important enough so nobody has done anything...